test/lint.js

```diff
@@ -54,7 +51,7 @@ function load(...files) {
 
         const console_error = console.error;
         console.error = (...args) => {
-          spinner.stream = process.stderr;
+          spinner['stream'] = process.stderr;
```

Comment:
error TS2339: Property 'stream' does not exist on type 'Ora'.

This is because `Ora.stream` is an internal property that is not actually part of the public API.

---

See also: https://github.com/PrismJS/prism/pull/1796#pullrequestreview-218108563